client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -113,7 +113,7 @@ const CartItem = React.createClass( {
 		} else if ( getIncludedDomain( this.props.cartItem ) ) {
 			info = getIncludedDomain( this.props.cartItem );
 		} else if ( isTheme( this.props.cartItem ) ) {
-			info = this.props.selectedSite.domain;
+			info = this.props.selectedSite && this.props.selectedSite.domain;
```

Comment:
`If it's a theme -> there's no meta` I'm not sure how can I come to this conclusion, could you elaborate please? =)